core/src/test/java/io/github/tcdl/msb/config/ServiceDetailsTest.java

```diff
@@ -0,0 +1,61 @@
+package io.github.tcdl.msb.config;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import io.github.tcdl.msb.api.exception.ConfigurationException;
+
+import org.junit.Test;
+
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+
+public class ServiceDetailsTest {
+    final String name = "test_msb";
+    final String version = "1.0.1";
+    final String instanceId = "msbd06a-ed59-4a39-9f95-811c5fb6ab87";
+
+    @Test
+    public void testServiceDetailsAll() {
+        String serviceDetailsConfigStr = "serviceDetails = {name = \"" + name + "\", version = \"" + version + "\", instanceId = \"" + instanceId + "\"}";
```

Comment:
I would suggest using StringBuilder instead of string concatenation for readability reasons